extension/popup.html

```diff
@@ -12,34 +12,59 @@
-          </div>
-        </main>
-      </div>
-      <main class="signed-in-panel hidden">
-        <alias-list>
-          <alias-list-header>
-            <span class="alias-list-header-title">Aliases</span>
-            <span class="click-to-copy-list-header">Click to copy</span>
-          </alias-list-header>
-        </alias-list>
-        <alias-creation>
-          <div class="gradient-button-wrapper">
-            <button class="create-new-alias gradient-button" type="submit" value="Generate new relay address" data-event-label="Create New Relay Alias - No existing aliases">
-              Create New Relay Alias
-            </button>
-          </div>
-        </alias-creation>
-        <footer>
-          <p class="ff-Met aliases-remaining text-center remaining-plural">You have <span class="num-aliases-remaining"></span> remaining free <span class="alias-text">aliases</span></p>
-          <a class="button new-tab close-popup-after-click footer-button" href="http://127.0.0.1:8000/accounts/profile?utm_source=addon&utm_medium=popup"></a>
+
+      <settings>
+        <div class="settings-header">
+          <button class="close-settings settings-toggle" title="Return to main panel"></button>
+          <h1 class="settings-hl">Settings</h1>
+        </div>
+        <ul class="settings-list">
+          <li class="setting">
+            <span class="setting-label flex-col">Show Relay icon in email fields on websites</span>
+            <button title="" data-icon-visibility-option="disable-input-icon" class="toggle-icon-in-page-visibility"></button>
+          </li>
+          <li class="setting flex-col">
+            <a href="https://addons.mozilla.org/firefox/addon/private-relay/?utm_source=fx-relay-addon&utm_medium=popup&utm_campaign=beta" class="setting-label close-popup-after-click">Leave Feedback</a>
+          </li>
+        </ul>
+      </settings>
+
+      <sign-up-panel class="sign-up-panel hidden">
+        <span class="ff-Met welcome">Welcome</span>
+        <p class="intro">
+          Get started by signing in with the Firefox account you used to join the Relay waitlist.
+        </p>
+        <a class="button sign-in-btn blue-primary-btn close-popup-after-click" href="http://127.0.0.1:8000/accounts/fxa/login/?process=login">
+          Continue
+        </a>
+        <div class="join-waitlist-wrapper">
+          <span class="">Curious about Private Relay?</span>
+          <a href="http://127.0.0.1:8000/accounts/fxa/login/?process=login" class="join-waitlist-link">Sign up for the waitlist with your Firefox account.</a>
+        </div>
+      </sign-up-panel>
+
+      <main-panel class="signed-in-panel hidden">
+          <p class="ff-Met aliases-remaining text-center remaining-plural">You have <span class="num-aliases-remaining"></span><span class="ff-Met max-num-aliases"></span></span> relay addresses remaining</p>
```

Comment:
this _might_ have an extra `</span>`. I count 2 opening but 3 closing.

Reply:
Confirmed w/ `prettier`:

```sh
npx prettier 'extension/**/*.html' --write

extension/first-run.html 58ms
extension/popup.html
[error] extension/popup.html: SyntaxError: Unexpected closing tag "span". It may happen when the tag has already been closed by another tag. For more info see https://www.w3.org/TR/html5/syntax.html#closing-elements-that-have-implied-end-tags (49:172)
[error]   47 |
[error]   48 |       <main-panel class="signed-in-panel hidden">
[error] > 49 |           <p class="ff-Met aliases-remaining text-center remaining-plural">You have <span class="num-aliases-remaining"></span><span class="ff-Met max-num-aliases"></span></span> relay addresses remaining</p>
[error]      |                                                                                                                                                                            ^
[error]   50 |           <onboarding-panel>
[error]   51 |             <img class="onboarding-img">
[error]   52 |             <h1 class="onboarding-h1"></h1>
```